basic_PHP/homework.php

```diff
@@ -0,0 +1,65 @@
+<?php 
+
+// $a = array(6, 2, 5, 10, 11, 1, 17, 20);
+// sort($a);
+// echo "Min=".$a[0]."<br>";
+// rsort($a);
+// echo "Max=".$a[0];
+
+//--------------------Homework 1------------------------
+$sum = 0;
+$number = 0;
+while($number <= 50) {
+    if($number % 2 == 1) $sum +=$number;
+    $number++;
+};
+echo "the sum of the odd numbers between 0 to 50: ".$sum."<br>";
+
+///////
+
+//--------------------Homework 2------------------------
+function factorialOfNumber($n)
+{
+    $result = 1;
+  // Write your source code here
+    for($i = 1; $i <= $n; $i++) {
+        $result *= $i;
+    }
+    return $result;
+}
+
+echo factorialOfNumber(10)."<br>";
+
+//////
+
+//--------------------Homework 3------------------------
+
+function check_PrimeNumber($inputNumber){

```

Comment:
![image](https://user-images.githubusercontent.com/76993901/130816685-3e98a420-88c2-4d25-9705-9a5960854008.png)
Please use space,  and {} for if .

Please research and add other method as type result same but bester